packages/docusaurus/src/webpack/base.ts

```diff
@@ -108,6 +108,7 @@ export function createBaseConfig(
       chunkFilename: isProd
         ? 'assets/js/[name].[contenthash:8].js'
         : '[name].js',
+      assetModuleFilename: 'assets/[name]-[hash][ext]',
```

Comment:
```suggestion
      assetModuleFilename: 'assets/[name]-[contenthash][ext]',
```
Usually contenthash gives better caching results.

Reply:
Yep, prefer to use `contenthash`